### ISSUE TITLE
Serial Monitor takes visibility from Output view during upload

### ISSUE THREAD

**Issue by per1234** [arduino/arduino-ide#1096]
### Describe the problem

When uploading to a native USB board while **Serial Monitor** is open, the "**Serial Monitor**" view takes visibility from the "**Output**" view twice during the upload:

![sm-focus](https://user-images.githubusercontent.com/8572152/174617761-ecb09b3b-ad90-4176-afc6-ec083bc88494.gif)

🐛 Users who want to monitor the upload process via the "**Output**" view will find it inconvenient to lose sight of it in the middle of the process, and then have to select the view once more after every upload.

### To reproduce

1. Connect a native USB board (e.g., [Leonardo](https://store.arduino.cc/products/arduino-leonardo-with-headers), [Micro](https://store.arduino.cc/products/arduino-micro), [MKR boards](https://store.arduino.cc/collections/mkr-family), [Nano 33 IoT](https://store.arduino.cc/products/arduino-nano-33-iot), [Nano 33 BLE](https://store.arduino.cc/products/arduino-nano-33-ble)) to your computer.
1. Select the board and port in the Arduino IDE.
1. Open the "**Serial Monitor**" view.
1. Select the "**Output**" view.
1. Click the "**Upload**" button.

🐛 The view visible in the bottom panel is switched multiple times, as follows:

1. "**Output**" view
1. "**Serial Monitor**" view
1. "**Output**" view
1. "**Serial Monitor**" view

Note that the view that is visible after the process finishes does not match the user's selected focus.

### Expected behavior

The "**Output**" view retains visibility throughout and after the upload process.

### Arduino IDE version

2.0.0-rc7-snapshot-a54d7c8

### Operating system

Windows

I am not able to reproduce the issue on my Linux machine.

### Operating system version

10

### Additional context

If the user had the "**Serial Monitor**" view selected at the time the upload process was triggered, it would be reasonable to return that view to visibility at the end of the upload process (as was requested [here](https://forum.arduino.cc/t/arduinoide-2-0-0-rc6-wemos-serial-monitor/999739)).

However, even in this case the switch from the "**Output**" view to the "**Serial Monitor**" view and then back to the "**Output**" view in the middle of the upload process is surely a poor user experience.

### Issue checklist

- [X] I searched for previous reports in [the issue tracker](https://github.com/arduino/arduino-ide/issues?q=)
- [X] I verified the problem still occurs when using the latest [nightly build](https://github.com/arduino/arduino-ide#nightly-builds)
- [X] My report contains all necessary details

**Comment by intoku** [comment]
yes, this is really annoying while coding. Hopefully this can be fixed soon
